tably.py

```diff
@@ -67,10 +67,13 @@ def __init__(self, args):
         self.preamble = args.preamble
         self.sep = get_sep(args.sep)
         self.units = args.units
-        self.no_escape = args.no_escape
         self.fragment = args.fragment
         self.fragment_skip_header = args.fragment_skip_header
         self.replace = args.replace
+        if args.no_escape:
```

Comment:
Nice idea!

I would write this as a ternary operator with no-op lambda, so there's no need to define `do_not_escape` function which does nothing:

```python
self.tex_str = escape if not args.no_escape else lambda x: x
```

Reply:
OK, code updated.